eng/pipelines/pull-request-common.yml

```diff
@@ -60,7 +60,7 @@ steps:
     displayName: E2E Tests
     # Skipping the e2e test on the prepare publish branch as there is a bug in the npm version installed on node 16
     # causing npx to try to download the unpublished version of the cli.
-    condition: eq('${{ parameters.nodeVersion }}', '18.x')
+    condition: ne('${{ parameters.nodeVersion }}', '16.x')
     # condition: and(not(startsWith(variables['System.PullRequest.SourceBranch'], 'publish/')), not(eq('${{ parameters.nodeVersion }}', '16.x')))
```

Comment:
while you're here the comment doesn't seem to quite match. Can you update the comment to not mention "preprare publish branch" and remove the commented out condition?